app/styles/_navbar-vertical.less

```diff
@@ -8,11 +8,10 @@
   background-repeat: no-repeat;
   // Max logo dimensions width 196px / height 40px
   height: @navbar-os-header-height-mobile - 1; // 1px border-bottom on navbar-os-mobile
-  width: 196px;
-  @media(min-width: @screen-sm-min) {
+  width: 191px;
+  // @media(min-width: @screen-sm-min) {
```

Comment:
Did you mean to leave this commented out?

Reply:
Yeah as it serves as documentation of sorts since the image used to/can get bigger as the viewport gets bigger.  I debated removing it, but opted to keep it since we don't have this documented anywhere...